install.sh

```diff
@@ -101,6 +101,14 @@ if [[ $1 == "cobc" ]]; then
   cd .. 
 fi
 
+if [[ $1 == "linux" ]]; then
+  cd include-what-you-use
+  mkdir build && cd build
+  cmake -G "Unix Makefiles" -DCMAKE_PREFIX_PATH=/usr/lib/llvm-15 ../
+  sudo make install
+  cd ../..
+fi
```

Comment:
Why is it only possible to install this for Linux? We have many source files that are only built for the COBC target, and I want to use IWYU for those as well. Or do I misunderstand this?

Also, please use the canonical form for the CMake commands:

~~~shell
cmake -S . -B build -G "Unix Makefiles" -DCMAKE_PREFIX_PATH=/usr/lib/llvm-15
sudo cmake --install build
~~~

Also, also, why specifically specify "Unix Makefiles"? IIRC, this is the default generator and if we somehow changed that to Ninja it should still work.

Reply:
> Why is it only possible to install this for Linux? We have many source files that are only built for the COBC target, and I want to use IWYU for those as well. Or do I misunderstand this?

Because the full image is based on the linux-x86 one, and we only need to build IWYU once. This is done when building the linux-x86 image, and the full image just inherits it.

The CMake commands and Unix Makefiles as build system comes from the installation documentation, but they can indeed be changed.

Reply:
Ah, ok, that makes sense.

Regarding the CMake commands I would have not specified any generator, just like for all the other libraries, but this way it is still fine.